src/api/app/controllers/webui/project_controller.rb

```diff
@@ -894,19 +894,20 @@ def remove_path_from_target
     redirect_to :action => :repositories, :project => @project
   end
 
-  def move_repo(direction)
+  def move_path_up
```

Comment:
In this case, I thought that removing the duplication causes the code to be harder to understand. But I guess it does not need to be. I can give it a try cleaning it up.